platforms/software/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationSpec.groovy

```diff
@@ -1544,36 +1544,6 @@ class DefaultConfigurationSpec extends Specification implements InspectableConfi
         t.message == "Mutation of attributes is not allowed"
     }
 
-    def dumpString() {
```

Comment:
This test is tedious to maintain. 
It tests the dump test fixture more than anything, which itself is not used anywhere.